docs/source/index.rst

```diff
@@ -71,8 +71,8 @@ and run a simple client/server example:
 Documentation
 *************
 
-*  API doc: `HTML <api/v1.6/html>`_ `PDF <api/v1.6/ucx.pdf>`_
-* `Examples <https://github.com/openucx/ucx/tree/v1.6.x/test/examples>`_
+*  API doc: `HTML <http://openucx.github.io/ucx/api/v1.7/html/index.html>`_ `PDF <http://openucx.github.io/ucx/api/v1.7/ucx-v1.7.pdf>`_
```

Comment:
@yosefe we really should change all the numbering to "latest" in all places with the exception of news